ansible/roles/iroha-docker/tasks/config-gen.yml

```diff
@@ -36,28 +36,26 @@
 
   - block:
 
-      - name: Generate genesis.block 1
-        set_fact:
-          iroha_new_genesis_block: "{{ iroha_genesis_block | from_json }}"
-
-      - name: Generate genesis.block 2
-        set_fact:
-          iroha_genesis_block_commands: "{{ (iroha_genesis_block_commands | default([])) + [item] }}"
-        loop: "{{ iroha_new_genesis_block['block_v1']['payload']['transactions'][0]['payload']['reducedPayload']['commands'] }}"
-        when: '"addPeer" not in item'
-
-      - name: Generate genesis.block 3
-        set_fact:
-          iroha_genesis_block_commands: "{{ iroha_genesis_block_commands + [{'addPeer': {'peer': {'address': item.hostname, 'peerKey': item.pub_key}}}] }}"
-        loop: "{{ iroha_all_new_nodes }}"
+      - name: Format existing genesis block
+        docker_container:
+          image: "iroha-docker-iroha-python"
```

Comment:
can you set image as variable, because some times it is not possible to build this image (pip not working, no internet), one may use local image which is in local registry.